django_netjsongraph/api/generics.py

```diff
@@ -38,7 +54,7 @@ class BaseReceiveTopologyView(APIView):
     parser_classes = (TextParser,)
 
     def post(self, request, pk, format=None):
-        topology = get_object_or_404(self.model, pk, strategy='receive')
+        topology = get_object_or_404(self.topology_model, pk, strategy='receive')
```

Comment:
I don't understand: why do we need to change this line? I think we can keep `self.model` because we don't need to handle different models in this view